src/rtc/h5_prep.py

```diff
@@ -27,6 +27,106 @@
 
 logger = logging.getLogger('rtc_s1')
 
+LAYER_NAME_VV = 'VV'
+LAYER_NAME_VH = 'VH'
+LAYER_NAME_LAYOVER_SHADOW_MASK = 'layover_shadow_mask'
+LAYER_NAME_RTC_ANF_GAMMA0_TO_BETA0 = 'rtc_anf_gamma0_to_beta0'
+LAYER_NAME_RTC_ANF_GAMMA0_TO_SIGMA0 = 'rtc_anf_gamma0_to_sigma0'
+LAYER_NAME_RTC_ANF_SIGMA0_TO_BETA0 = 'rtc_anf_sigma0_to_beta0'
+LAYER_NAME_RTC_ANF_BETA0_TO_BETA0 = 'rtc_anf_beta0_to_beta0'
+LAYER_NAME_NUMBER_OF_LOOKS = 'number_of_looks'
+LAYER_NAME_INCIDENCE_ANGLE = 'incidence_angle'
+LAYER_NAME_LOCAL_INCIDENCE_ANGLE = 'local_incidence_angle'
+LAYER_NAME_PROJECTION_ANGLE = 'projection_angle'
+LAYER_NAME_RTC_ANF_PSI = 'rtc_anf_psi'
+LAYER_NAME_RANGE_SLOPE = 'range_slope'
+LAYER_NAME_DEM = 'dem'
```

Comment:
Thank you, @seongsujeong . What would be the keys for that dictionary? You need to define them somewhere. These constants define the keys that are used in the dictionaries defined below.

Reply:
I see. So it look like those constants are used as keys in the following dictionaries (e.g. `layer_hdf5_dict`, `layer_names_dict`), as well as layer names in `rtc_single_jb.py` and etc.
That being case, I think the dictionary name is `LAYER_NAME_DICT`, and keys are `VV`, `VH`, `LAYER_SHADOW_MASK` and so on.

The point that I am suggesting here is to bundle the layer name constants to organize them better.

If you prefer, I think `SimpleNamespace` instead of `dictionary` would be okay to bundle the layer names here.

Reply:
Sounds good. Let's work on that after merging this PR.

Reply:
Thank you, @seongsujeong . Please. let me know if you have any other comments or concerns.